src/accelerate/utils/dataclasses.py

```diff
@@ -1339,5 +1339,4 @@ class IntelPyTorchExtensionPlugin:
     def set_mixed_precision(self, mixed_precision):
         if mixed_precision == "fp16":
             raise ValueError("Tried to use `fp16` but it is not supported on cpu or xpu")
-        elif mixed_precision == "bf16":
-            self.dtype = torch.bfloat16
+        self.dtype = mixed_precision
```

Comment:
Said this before on other PRs, this `dtype` should not be used (and should be removed ideally).